vin/app/controllers/vin/notes_controller.rb

```diff
@@ -1,77 +0,0 @@
-class Vin::NotesController < ApplicationController
-respond_to :json
-
-  # GET /vin/sub/:uid/shipments/:sid/notes
-  def index
-
-    @subscriber = Subscriber.find(params[:sub_id])
-
-    @package = @subscriber.shipments.find(params[:shipment_id]).package
-
-    notes = @package.note
-
-    render_result({ notes: [notes.to_s] })
-  end
-
-  # POST /vin/sub/:uid/shipments/:sid/notes
-  def create
-
-    @subscriber = Subscriber.find(params[:sub_id])
-
-    @package = @subscriber.shipments.find(params[:shipment_id]).package
-
-    note = params[:content]
-
-    status = @package.update_attribute(:note, note)
-
-    render_result({ errors: [status ? "" : "Problem updating note"] })
```

Comment:
Usually style guides advise against the ternary operator. It's nothing but a convoluted `if` statement